doc/sphinx_source/using/accounts.rst

```diff
@@ -0,0 +1,81 @@
+::
+
+  Loaded module information:
+    #eggdroptest        :   2 members, enforcing "+tn" (greet)
+    Channels: #eggdroptest (need ops)
+    Account tracking: Best-effort (Missing capabilities: extended-join, see .status all for details)      <---- This line
+    Online as: Eggdrop (Eggdrop)
+
+Determining if a Server Supports Account Capabilities
+-----------------------------------------------------
+A server announces the capabilities it supports via a CAP request. If you have Tcl enabled on the partyline (or via a raw message from a client), you can send `.tcl cap ls` and see if the extended-join and account-notify capabilities are supported by the server. If they are not listed, the server does not support it.
+
+A server announces if it supports WHOX via its ISUPPORT (005) announcement. If you have Tcl enabled on the partyline, you can send `.tcl issupport isset WHOX` and if it returns '1', WHOX is supported by the server.
+
+Best-Effort Account Tracking
+----------------------------
+If a server only supports some, but not all, of the required capabilities, Eggdrop will switch to 'best effort' account tracking. This means Eggdrop will update account statuses whenever it sees account information, but Eggdrop cannnot guarantee the accuracy of account association at any given time.
+
+If a server does not support extended-join, Eggdrop will not be able to determine the account associated with a user when they join. Eggdrop can update this information by sending a WHOX to the server.
+
+If a server does not support account-notify, Eggdrop will not be able to determine the account assoicated with a user if they authenticate/deauthenticate from their account after joining a channel. Eggdrop can update this information by sending a WHOX to the server.
+
+If a server does not support WHOX, Eggdrop will not be able to determine the accounts associated with users already on a channel before Eggdrop joined. There is no reliable way to update this information.
+
+One workaround to significantly increase the accuracy of account tracking for scripts in a 'best effort' scenario would be to issue a WHOX query (assuming the server supports it), wait for the reply from the eserver, and then query for the account information.
+
+account-tag
+^^^^^^^^^^^
+One supplementary capability that can assist a best-effort account tracking scenario is the IRCv3-defined `account-tag capability <https://ircv3.net/specs/extensions/account-tag>`_. The account-tag capability attaches a tag with the account name associated with the user sending a command. Enabling this capability allows Eggdrop to update its account tracking every time a user talks in channel, sets a mode, sends a kick, etc. While still not able to offer the same level of accuracy as enabling the "main three" account tracking features, it can increase the accuracy and can help with scripts that react to user commands/messages.
```

Comment:
Maybe mention `[getaccount $nick $chan]` specifically working in bind callback procs